packages/ui/src/PasswordField.module.scss

```diff
@@ -0,0 +1,20 @@
+@use '../styles/constants' as c;
+
+$buttonMargin: c.$grid * 2;
+$inputMargin: c.$iconSizeSmall + $buttonMargin * 2;
+
+.inputContainer {
+  position: relative;
+
+  > input {
+    padding-right: 0;
+    margin-right: $inputMargin;
+  }
+
+  .toggle {
+    position: absolute;
+    right: $buttonMargin;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
nit: I'm thinking whether it wouldn't be more readable to use `margin: auto, top: 0, bottom: 0` and specify an explicit height instead of top50% and translateY.

Reply:
Don't you think that transform-translate is a more or less standard way to centralize an absolutely positioned element? Moreover, it would work with any input height.

Reply:
Both are fine, thus => `nit`, maybe we can use grid for these kind of things? It's supported in IE11 with prefixes and solves a lot of these. WDYT?

Reply:
caniuse says IE11 supports only the older version of the spec https://caniuse.com/css-grid
I am kind of concerned with all possible issues. Have you tried using it for IE11 yourself?

Reply:
It is indeed limited and there is not a full support and only with prefixes, but for the basic usage like making a simple grid and position elements, I think it's totally doable also for production. I remember we used it for enterprise product while IE11 was required there and there were no complaints at all. But we were not overusing it that is true :)